src/main/kotlin/racingcar/model/CarRacingGame.kt

```diff
@@ -0,0 +1,45 @@
+package racingcar.model
+
+import racingcar.RandomCountGenerator
+
+/**
+ * 자동차 경주
+ */
+class CarRacingGame(
+    private val numberOfGame: Int,
+    private val carRacers: List<CarRacer>
+) {
+
+    private val resultGames: MutableMap<Int, String> = mutableMapOf()
+
+    fun start() {
+        for (i in 1..numberOfGame) {
+            startGame()
+            saveResult(i)
+        }
+    }
```

Comment:
- (MutableMap 을 안쓴다면) 아래와 같은 표현도 가능할거 같네요.
```suggestion
    fun start() {
        repeat(numberOfGame) { 
            startGame()
            saveResult()
        }
    }
```

Reply:
Map 은 빼고 새롭게 다시 로직을 구현합니다!